Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
+  However, it is not the case that a given characteristic necessarily corresponds with at most one value, univocally. It could very well be the case that the red         characteristic of Washington Red Delicious Apples might also have another corresponding value in a red colored car of the same hue. This means that correspondence     between a characteristic and a bearer is not necessarily functional, i.e. has a binary value between 0 and 1 at most. 
+  A characteristic can have a one to one relationship to a bearer, a one to many relationship, a many to many relationship, or a many to one relationship. 
+  What could be the explanation of why this object property was set to having the functional characteristic? Upon further analysis, one can find a note in the           ontology:
+  “Note that this relation was previously called "inheres in", but was changed to be called "characteristic of" because BFO2 uses "inheres in" in a more restricted       fashion. This relation differs from BFO2:inheres_in in two respects: (1) it does not impose a range constraint, and thus it allows qualities of processes, as well as   of information entities, whereas BFO2 restricts inheres_in to only apply to independent continuants (2) it is declared functional, i.e. something can only be a         characteristic of one thing.”
+  (2) states: “it is declared, i.e. something can only be a characteristic of one thing.” What one might infer from this declaration is that this was done for           pragmatic purposes in order to elicit preferred inference relations without having to deal with the burden of unwanted inferences to irrelevant elements. So for       example, by declaring a functional constraint on “characteristic of” the ontologist could make pragmatic use of a delimited ontology that always garners a one to one   relation, rather than a one to many, and so on. As a result, if the reasoner is run the functional constraint set will restrict the domain to one to one functional     relations only. Thereby granting the conditions for a more specific, restricted ontology.
+  However, I must make a case for unlisting the functional characteristic. If I was an ontologist seeking to construct a knowledge base outside of a delimited domain     of one to one functional relationships and wanted to represent the class of all red things, then I ought to unlist the functional characteristic so that the object     property relation “characteristic of” could represent and infer a one to many relationship, which is not functional by definition, i.e. 0..1..X.
```

Comment:
We have the universal redness, and it has many instances in red things. Every instance of redness, like a redness trope, has one-to-one correspondence with a particular red thing. In my eyes this setting is rather elegant and useful in tracking an instantiation of a universal by an individual.